use/events.js

```diff
@@ -196,6 +200,8 @@ export const useEvents = () => {
       label: t('event.startDate'),
       simple: true,
       onChange: updateEndDate,
+      validation: (val) => !!val,
```

Comment:
would this work if val = '  '. might have to trim() the val to remove whitespace.

Reply:
validation functions for specific fields is not in the scope of this issue, it's just an example